flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/catalog/JdbcCatalogUtils.java

```diff
@@ -49,7 +51,12 @@ public static AbstractJdbcCatalog createCatalog(
             String baseUrl) {
         JdbcDialect dialect = JdbcDialectLoader.load(baseUrl, userClassLoader);
 
-        if (dialect instanceof PostgresDialect) {
+        if (dialect
+                instanceof
+                CrateDBDialect) { // Should precede PostgresDialect as it inherits from it
```

Comment:
I didn't follow this PR since beginning, so may miss something...
However it seems that now 'CrateDBDialect` extends `AbstractPostgresCompatible` [1] which makes this comment invalid

[1] https://github.com/apache/flink-connector-jdbc/pull/29/files#diff-b0c1a7873f630cdde5124db2fdc159cdf381e2ffb53ea96c30fb941506459177R27

Reply:
makes sense, fixed.